phpBB/phpbb/filesystem/filesystem.php

```diff
@@ -162,26 +162,7 @@ public function chown($files, $user, $recursive = false)
 	 */
```

Comment:
Should i move it to the helper class too?
Also i think i will have to move the helper class to filesystem , it makes no sense in storage